prototypes/ScheduledMergesTestQLS.hs

```diff
@@ -35,17 +35,17 @@ prop_LSM = Lockstep.runActions (Proxy :: Proxy Model)
 
 type ModelLSM = Int
 
-newtype Model = Model { mlsms :: Map ModelLSM (Map Key Value) }
+newtype Model = Model { mlsms :: Map ModelLSM (Map Key (Value, Maybe Blob)) }
   deriving stock (Show)
```

Comment:
I'm okay with including blobs, though I'd like to get some clarification here why we're adding blob support. Is it an aspect we need to prototype?

For scheduling, it does not matter whether blobs are supported, because a value + blob is isomorphic to just a value (that includes the blob as part of the value).

For unions, blobs are combined differently than values, so in that sense it may be useful. Though you could also argue that the important aspect of unions to prototype is proper bracketing, and I suspect that does not rely much on whether updates include mupserts and/or blobs.

Does that make sense?

Reply:
I agree that blobs aren't the most interesting things to model here and I wouldn't have added them from scratch. However, they were already supported in the prototype, but only through `update`, not `insert`. Also, they weren't tested (and not part of the QLS model). So I just wanted to make it more consistent.